.github/workflows/agent-build-refactored.yml

```diff
@@ -23,9 +25,10 @@ jobs:
   pre_job:
     runs-on: ubuntu-20.04
     outputs:
-      should_skip: ${{ steps.skip_check.outputs.should_skip }}
```

Comment:
Out of curiosity, why was this removed / no longer necessary?

Reply:
Because for this workflow I just left `opened`  event for the `pull_request` trigger type, https://github.com/scalyr/scalyr-agent-2/blob/0896974089c106ac251e7f731f27c9d6463a7a98/.github/workflows/agent-build-refactored.yml#L4-L7
 so there should not be any "double triggering" from both the `push` and `pull_request` when you push new commits to already opened pull request.